### DISCUSSION TITLE
How can I solve this?

### DISCUSSION THREAD

**Discussion by RazorDarkness01** [TownyAdvanced/Towny#5963] [Q&A] [answered]
![Captura](https://user-images.githubusercontent.com/107450783/173482969-e85152be-f99d-44d9-b403-b0ac3466bda3.PNG)

**Comment by ItsJuls** [comment]
its probably in the config that you have to change it

**Reply by RazorDarkness01** [reply]
What is it that I have to change?

**Comment by ItsJuls** [comment]
# maximum number of plots any single resident can own
  max_plots_per_resident: '100'

change that i think

**Reply by RazorDarkness01** [reply]
It doesn't work, idk why

**Reply by ItsJuls** [reply]
are you opped?
can you come to the towny discord?

**Reply by RazorDarkness01** [reply]
I'm, the owner, yes I'm opped.
Can you pass me the link please?

**Reply by ItsJuls** [reply]
https://discord.gg/NMBYyuSA

**Comment by LlmDl** [comment, marked as answer]
For some reason PowerRanks causes this, probably because of an incomplete Vault implementation (I have not looked into why.)
